pyaedt/edb_core/EDB_Data.py

```diff
@@ -61,68 +62,68 @@ def messenger(self):
 
     @property
     def name(self):
-        """
+        """Retrieve the layer name.
 
         Returns
         -------
         str
-            Layer Name
+            Name of the layer.
         """
         if not self._name:
             self._name = self._layer.GetName()
         return self._name
 
     @property
     def id(self):
-        """
+        """Retrieve the layer ID.
 
         Returns
         -------
         int
-            Layer ID
+            ID of the layer.
         """
         if not self._id:
             self._id = self._layer.GetLayerId()
         return self._id
 
     @property
     def layer_type(self):
-        """
-
-        Parameters
-        ----------
-        self
+        """Retrieve the layer type.
 
         Returns
         -------
         int
-            Layer Type
+            Type of the layer.
         """
         if not self._layer_type:
             self._layer_type = self._layer.GetLayerType()
         return self._layer_type
 
     @layer_type.setter
     def layer_type(self, value):
```

Comment:
Setters don't have to be documented.  All documentation for a setter should be placed where it's defined in `@property`.  Reason for this is when our documentation is built, it's only going to print out what's listed in `@property`, so anything we write in the setter is effectively invisible in our generated docs.

`@layer_type.setter` is pretty much self documenting, so we can just skip the docstrings for all setters.

One note.  When documenting properties that also have setters, you'd document it with:

```
Retrieve or update the layer type.
```
Basically, we're exposing a `get/set` to the user with these methods, rather than just the `get` method.